BAZProjectC1/Models/Strings.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Strings.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 15/09/22.
+//
+
+import Foundation
+
+enum EndpointsList:String  {
+    case movieAPI = "movieAPI"
+    case imageResorce = "imageResorce"
```

Comment:
Nitpick: Hay un error de orografía en la palabra Resource.
```suggestion
    case imageResource = "imageResource"
```